src/gui/macutils/AppKitImpl.mm

```diff
@@ -118,6 +118,31 @@ - (void) userSwitchHandler:(NSNotification*) notification
     }
 }
 
+//
+// Add global event monitor
+//
+- (id) addGlobalMonitor:(NSEventMask) mask handler:(void (^)(NSEvent *)) handler
+{
+    return [NSEvent addGlobalMonitorForEventsMatchingMask:mask handler:handler];
+}
+
+//
+// Remove global event monitor
+//
+- (void) removeGlobalMonitor:(id) monitor
+{
+    [NSEvent removeMonitor:monitor];
+}
+
+//
+// Check if accessibility is enabled, may show an popup asking for permissions
+//
+- (bool) enableAccessibility
+{
+    NSDictionary *opts = @{static_cast<id>(kAXTrustedCheckOptionPrompt): @YES};
```

Comment:
NSDictionary* opts

There are more things like that in this file (and the next).